app/views/task_doer_sessions/sign_up.html.erb

```diff
@@ -0,0 +1,40 @@
+<h1>Sign up</h1>
+
+<form action="/post_task_doer" method="post">
```

Comment:
This needs to change to either a `get` to `/insert_task_doer` or the route needs to change to `post`.